pkg/policy/criteria/claim_test.go

```diff
@@ -40,10 +44,39 @@ allow:
 		require.Equal(t, A{false, A{}}, res["deny"])
 	})
 	t.Run("by session claim", func(t *testing.T) {
+		t.Parallel()
+
 		res, err := evaluate(t, `
 allow:
   and:
     - claim/family_name: Smith
+`,
+			[]*databroker.Record{
+				makeRecord(&session.Session{
+					Id:     "SESSION_ID",
+					UserId: "USER_ID",
+					Claims: map[string]*structpb.ListValue{
+						"family_name": {Values: []*structpb.Value{structpb.NewStringValue("Smith")}},
+					},
+				}),
+				makeRecord(&user.User{
+					Id:    "USER_ID",
+					Email: "test@example.com",
+				}),
+			},
+			Input{Session: InputSession{ID: "SESSION_ID"}})
+		require.NoError(t, err)
+		require.Equal(t, A{true, A{ReasonClaimOK}, M{}}, res["allow"])
+		require.Equal(t, A{false, A{}}, res["deny"])
+	})
+	t.Run("by session claim via has", func(t *testing.T) {
```

Comment:
We should also probably test the opposite case, e.g. if the value doesn't contain "Smith", it is not accepted